pootle/checks.py

```diff
@@ -0,0 +1,176 @@
+
+    errors = []
+
+    try:
+        queue = get_queue()
+        workers = Worker.all(queue.connection)
+    except Exception as e:
+        conn_settings = queue.connection.connection_pool.connection_kwargs
+        errors.append(checks.Critical(_("Could not connect to Redis (%s)") % (e),
+            hint=_("Make sure Redis is running on %(host)s:%(port)s") % (conn_settings),
+            id="pootle.C001",
+        ))
+    else:
+        if not workers or workers[0].stopped:
+            # We need to check we're not running manage.py rqworker right now..
+            import sys
+            if len(sys.argv) > 1 and sys.argv[1] in RQWORKER_WHITELIST:
+                errors.append(checks.Warning(_("No RQ Worker running."),
+                    hint=_("Run new workers with manage.py rqworker"),
+                    id="pootle.W001",
+                ))
+
+        redis_version = queue.connection.info()["redis_version"].split(".")
+        if tuple(int(x) for x in redis_version) < REDIS_MINIMUM_REQUIRED_VERSION:
+            errors.append(checks.Warning(_("Your version of Redis is too old."),
+                hint=_("Update your system's Redis server package"),
+                id="pootle.W002",
+            ))
+
+    return errors
+
+
+@checks.register()
+def test_settings(app_configs=None, **kwargs):
+    from django.conf import settings
+
+    errors = []
+
+    if "RedisCache" not in settings.CACHES.get("default", {}).get("BACKEND"):
+        errors.append(checks.Critical(_("Cache backend is not set to Redis."),
+            hint=_("Set default cache backend to django_redis.cache.RedisCache\n"
+                 "Current settings: %r") % (settings.CACHES.get("default")),
+                id="pootle.C005",
+        ))
+    else:
+        from django_redis import get_redis_connection
+
+        if not get_redis_connection():
+            errors.append(checks.Critical(_("Could not initiate a Redis cache connection"),
+                hint=_("Double-check your CACHES settings"),
+                id="pootle.C004",
+            ))
+
+    if settings.DEBUG:
+        errors.append(checks.Warning(_("DEBUG mode is on. Do not do this in production!"),
+            hint=_("Set DEBUG = False in Pootle settings"),
+            id="pootle.W005"
+        ))
+    elif "sqlite" in settings.DATABASES.get("default", {}).get("ENGINE"):
+        # We don't bother warning about sqlite in DEBUG mode.
+        errors.append(checks.Warning(_("The sqlite database backend is unsupported"),
+            hint=_("Set your default database engine to postgresql_psycopg2 or mysql"),
+            id="pootle.W006",
+        ))
+
+    if settings.SESSION_ENGINE.split(".")[-1] not in ("cache", "cached_db"):
+        errors.append(checks.Warning(_("Not using cached_db as session engine"),
+            hint=_("Set SESSION_ENGINE to django.contrib.sessions.backend.cached_db\n"
+                 "Current settings: %r") % (setting.SESSION_ENGINE),
```

Comment:
@jleclanche Please align the content as told in http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#continuing-long-statements